.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,37 @@
+// If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
+{
+	"name": "Refinebio API",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspace",
+
+	// Use 'settings' to set *default* container specific settings.json values on container create.
+	// You can edit these settings after create using File > Preferences > Settings > Remote.
```

Comment:
Well this is very ugly :(

Reply:
yeah - seems like github doesn't like comments in json files

Reply:
Well that's just because JSON doesn't support comments. It's not valid JSON but I guess if you wanna use JSON for configs people write special parsers that support it?

Reply:
It doesn't seem to be a problem for vscode, this is a configuration file for the IDE.